app/qml/map/MMMapController.qml

```diff
@@ -97,44 +97,53 @@ Item {
     updatePosition()
 
     switch ( state ) {
+      case "record": {
+        if ( __appSettings.autolockPosition ) { // center to GPS
+          if ( gpsStateGroup.state === "unavailable" ) {
+            __notificationModel.addError( "GPS currently unavailable." )
+          }
+          else {
+            root.centeredToGPS = true
```

Comment:
if we moved this out of the GPS availability check, does it mean the map would re-center as soon as gps position is actually available? If yes, that could be useful, but I'm OK with merging it as it is now!

Reply:
In this scenario, it would center even if the user decides to uncheck the "auto-lock" property in the recording settings.

Reply:
No, I mean like:
```qml
        if ( __appSettings.autolockPosition ) { // center to GPS
          root.centeredToGPS = true
          if ( gpsStateGroup.state === "unavailable" ) {
            __notificationModel.addError( "GPS currently unavailable." )
          }
          else {
            updatePosition()
          }
        }